reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -112,13 +112,46 @@ function renderLicensesTable (
   for (let i = 0; i < columnNames.length; i++)
     columnNames[i] = chalk.blueBright(columnNames[i])
 
+  const data = [
+    columnNames,
+    ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
+      .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
+  ]
+  let detailsColumnMaxWidth = 40
+  let packageColumnMaxWidth = 0
+  let licenseColumnMaxWidth = 0
+  if (opts.long) {
+    detailsColumnMaxWidth = data.slice(1).reduce((maxWidth, row) => {
+      const texts = row[2].split('\n')
+      const link = texts.find((line) => line.startsWith('https'))
+      // Use the package link to determine the width of the details column
+      const detailMaxWidth = link ? link.length : 0
+      const repeatCount = Math.max(0, texts.length - 1)
+      row[0] = `${row[0]}${'\n '.repeat(repeatCount)}` // Add extra lines to the package column
+      row[1] = `${row[1]}${'\n '.repeat(repeatCount)}` // Add extra lines to the license column
```

Comment:
isn't this the same?
```suggestion
      row[0] += '\n '.repeat(repeatCount) // Add extra lines to the package column
      row[1] += '\n '.repeat(repeatCount) // Add extra lines to the license column
```

Reply:
Yes.